File: Java/Advanced Conditional Statements/Exercise/ACS Exercise/src/Cinema.java
import java.util.Scanner;

public class Cinema {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String projectionType = input.nextLine();
        int r = Integer.parseInt(input.nextLine());
        int c = Integer.parseInt(input.nextLine());

        double earnings = 0;


        switch(projectionType){
            case "Normal":
                earnings = 7.50 * (r * c);
                break;
            case "Premiere":
                earnings = 12.00 * (r * c);
                break;
            case "Discount":
                earnings = 5 * (r * c);
                break;
        }

        System.out.printf("%.2f", earnings);
    }
}

File: Python/Basics/Conditional Statements Advanced/Exercise/Cinema.py
projectionType = input()
r = int(input())
c = int(input())
earnings = 0
ticketCost = 0

if projectionType == "Premiere":
    ticketCost = 12.00
elif projectionType == "Normal":
    ticketCost = 7.50
elif projectionType == "Discount":
    ticketCost = 5.00


totalSeats = r * c
earnings = ticketCost * totalSeats

print(f"{earnings:.2f} leva")